modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -21,13 +21,17 @@
  */
 
 #include "context-info-db.h"
+#include "scratch-buffers2.h"
+#include "apphook.h"
 #include <criterion/criterion.h>
 #include <criterion/parameterized.h>
 #include <stdio.h>
 #include <string.h>
 
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof((a)[0]))
 
+TestSuite(add_contextual_data, .init=app_startup, .fini=app_shutdown);
+
```

Comment:
I would prefer not to spread app_startup() into unit tests. I know there are some already, but I consider them excessive that increases coupling.